### PR TITLE
Add upvoters and downvoters tooltip

### PR DESCRIPTION
**Pull request by nclsHart**
Preview : 

voters-tooltip

### CODE REVIEW

File: app/application.js [hunk 1]
```diff
@@ -115,6 +115,20 @@ angular.module('app', ['config.app', 'emojify', '720kb.tooltips'])
...
 
+                mergeRequest.upvoters = [];
+                notes.forEach(function (note) {
+                    if (note.upvote) {
+                        mergeRequest.upvoters.push(note.author.name);
+                    }
+                });
+
+                mergeRequest.downvoters = [];
+                notes.forEach(function (note) {
```
Comment (notFloran) [inline app/application.js]: why two foreach ?

Comment (nclsHart) [inline app/application.js]: I really don't know why...

**Comment by notFloran** [comment]
:+1:

**Comment by ghost** [comment]
:+1:

**Pull request merged**